deeplabcut/utils/conversioncode.py

```diff
@@ -386,3 +397,187 @@ def guarantee_multiindex_rows(df):
 def robust_split_path(s):
     sep = "/" if "/" in s else "\\"
     return tuple(s.split(sep))
+
+def extract_frames_from_pkg_slp(file_path, base_output_dir, scorer='me'):
```

Comment:
suggest changing `extract_frames_from_pkg_slp` to `extract_frames_from_pkg_sleap`